CONTRIBUTING.md

```diff
@@ -127,6 +127,25 @@ reference GitHub issues that this commit **Closes**. In order to maintain a refe
 `fixes #<issue_number>` if it closes a related issue or `issue #<issue_number>`
 if it's a partial fix.
 
+## What should you do once your PR is merged?
```

Comment:
I might almost re-contextualize this section to be more broadly about syncing your fork; there are plenty of reasons to keep your fork in sync beyond your PR getting merged. What do you think? I am so happy you documented this though 🙂

Reply:
Yeah right, since I opened this PR simply with a view to help out new contributors, I most certainly didn't realize the scope of keeping the fork in sync. Thanks for pointing that out! 
With this point being made, I wonder if this section after re-contextualizing should be kept under CONTRIBUTING.md or README.md? What are your thoughts? Also, thanks a lot for the acknowledgement :)

Reply:
I think it's probably still fine in the Contributing doc since it deals with Git and your setup more-so than working on the code.

Reply:
> I think it's probably still fine in the Contributing doc since it deals with Git and your setup more-so than working on the code.

Yeah makes sense!